presto-kafka/pom.xml

```diff
@@ -188,6 +188,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Any specific reason for adding this dependency ?

Reply:
Otherwise building the module presto-kafka fails: 
[INFO] --- maven-dependency-plugin:3.1.2:analyze-only (default) @ presto-kafka ---
[WARNING] Used undeclared dependencies found:
[WARNING]    junit:junit:jar:4.12:test